packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -1,55 +1,61 @@
 import type { AbstractModel, DetachedModelConstructor } from '@hilla/form';
 import { Button } from '@hilla/react-components/Button.js';
 import { GridColumn } from '@hilla/react-components/GridColumn.js';
-import { HorizontalLayout } from '@hilla/react-components/HorizontalLayout.js';
-import { VerticalLayout } from '@hilla/react-components/VerticalLayout.js';
 import { type JSX, useState } from 'react';
 import { AutoCrudContext } from './autocrud-context.js';
 import DeleteButton from './autocrud-delete.js';
+import { AutoCrudDialog } from './autocrud-dialog';
 import { emptyItem, ExperimentalAutoForm } from './autoform.js';
 import { AutoGrid } from './autogrid.js';
 import type { CrudService } from './crud.js';
+import { useMediaQuery } from './media-query';
 import { getProperties } from './property-info.js';
 
 export type AutoCrudProps<TItem> = Readonly<{
   service: CrudService<TItem>;
   model: DetachedModelConstructor<AbstractModel<TItem>>;
   noDelete?: boolean;
-  header?: string;
 }>;
 
-export function ExperimentalAutoCrud<TItem>({ service, model, noDelete, header }: AutoCrudProps<TItem>): JSX.Element {
+export function ExperimentalAutoCrud<TItem>({ service, model, noDelete }: AutoCrudProps<TItem>): JSX.Element {
   const [item, setItem] = useState<TItem | typeof emptyItem | undefined>(undefined);
   const [refreshTrigger, setRefreshTrigger] = useState(0);
+  const fullScreen = useMediaQuery('(max-width: 600px), (max-height: 600px)');
```

Comment:
This is the same query that `vaadin-crud` uses:
https://github.com/vaadin/web-components/blob/735f5d90df1618c700bfc7be0d9c8ad0cd8436dc/packages/crud/src/vaadin-crud.js#L627-L629